code-challenges.test.js

```diff
@@ -32,12 +85,39 @@
 var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 // Expected output: [-9, 7, 9, 199]
 
+describe("odds", () => {
+    it("returns odds", () => {
+        expect(odds(fullArr1)).toEqual([-9, 7, 9, 199])
+    })
+})
+
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+describe("odds", () => {
+    it("returns odds", () => {
+        expect(odds(fullArr2)).toEqual([-823, 7, 23])
+    })
+})
+
```

Comment:
Again your test logic looks fantastic. Just need to wrap all this content together. 
```
describe("odds", () => {
    it("returns odds", () => {
        var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
        var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
        expect(odds(fullArr1)).toEqual([-9, 7, 9, 199])
        expect(odds(fullArr2)).toEqual([-823, 7, 23])
    })
})
```